### PR TITLE
[CB-116] Add ModifiedSupportSpecs predictor

### PR DESCRIPTION
**Pull request by EPecherkin**

### CODE REVIEW

File: lib/crystalball/predictor/modified_support_specs.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+module Crystalball
+  class Predictor
+    # Used with `predictor.use Crystalball::Predictor::ModifiedSupportSpecs.new`. Will find files that match passed regexp and
+    # return full spec files which uses matched support spec files. Perfectly works for shared_context and shared_examples.
+    class ModifiedSupportSpecs
+      include ::Crystalball::Predictor::Helpers::AffectedExamplesDetector
+
+      # @param [Regexp] support_spec_pattern - regexp to filter support specs files
+      def initialize(support_spec_pattern = %r{spec/support/.*\.rb\z})
```
Comment (pyromaniac) [inline lib/crystalball/predictor/modified_support_specs.rb:13]: Should not this default be somewhere in settings?

**Review by pyromaniac — APPROVED** [review]
LG if it works

**Review by jaimerson — APPROVED** [review]
LGTM

File: lib/crystalball/predictor/modified_support_specs.rb (hunk 1 above)
Comment (EPecherkin) [reply lib/crystalball/predictor/modified_support_specs.rb:13]: It is a common way to configure predictors right now. I think it makes sense to move it to config file later

**Pull request merged**